manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: mtkennerly.shawl
+PackageVersion: 1.5.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: shawl.exe
+  PortableCommandAlias: shawl
+Installers:
```

Comment:
```suggestion
Commands: --help
Installers:
```

Reply:
Thanks! I've just pushed the update (had to use an array).

Reply:
I've reverted this change based on https://github.com/microsoft/winget-pkgs/pull/143761#issuecomment-1989004315 .